src/db.rs

```diff
@@ -1768,9 +1768,10 @@ impl ReadOptions {
         }
     }
 
+    /// set_iterate_upper_bound sets the upper bound for an iterator, and the upper bound itself
```

Comment:
IMO `set_iterate_upper_bound` is redundant here, isn't it ?

Reply:
Yes, I would say it is.

Reply:
thanks, I would fix it.